docs/mesos/ManualInstallNetmodules.md

```diff
@@ -0,0 +1,59 @@
+# Manually Install Mesos + Netmodules
+This guide will walk you through building and installing Mesos + Net-Modules from source.
+
+## Background
+Netmodules and Mesos both make use of the `protobuf` library. To function correctly, Mesos and Netmodules must be built with identical compilations of `protobuf`. A standard Mesos installation will include bundled versions of this library, so we'll compile Mesos with an unbundled to ensure that netmodules is using precisely the same library as Mesos.
+
+## 1. Install Dependencies
+You can manually compile those libraries yourself, but we'll just install protobuf-devel from the `epel-release` repositories:
+```
+sudo yum install -y protobuf-devel protobuf-python boost-devel glog-devel
+```
+>A note on Glog: At the time of this writing, the `glog-devel` rpm package does not satisfy Mesos' glog dependency. If you find this to be true, try manually compiling and installing glog instead.
+
+Install the picojson headers:
+
+    wget https://raw.githubusercontent.com/kazuho/picojson/v1.3.0/picojson.h -O /usr/local/include/picojson.h
+
+## 2. Build and Install Mesos
+Next we'll follow the standard Mesos installation instructions, but pass a few flags to configure to use our installed libraries instead of the mesos bundled ones:
+
+```
+# Download Mesos source
+$ git clone git://git.apache.org/mesos.git
+$ cd mesos
+
+# Configure and build.
+$ ./bootstrap
+$ mkdir build
+$ cd build
+$ ../configure --with-protobuf=/usr --with-boost=/usr --with-glog=/usr
+$ make
+$ sudo make install
+```
+
+## 3. Build and Install Netmodules
+```
+# Download netmodules source
+$ git clone https://github.com/mesosphere/net-modules.git
+$ cd net-modules/isolator
+
+# Configure and build
+$ ./bootstrap
+$ mkdir build
+$ cd build
+$ ../configure --with-mesos=/usr/local --with-protobuf=/usr
+$ make
+$ sudo make install
+```
+
+## 4. Launch Mesos-Slave
+Must provide ETCD_AUTHORITY variable here so when mesos uses the calico_mesos plugin
+it can succesfully communicate with etcd.
+```
+sudo ETCD_AUTHORITY=172.17.0.3:4001 /usr/local/sbin/mesos-slave \
+--master=<MASTER-IP:PORT> \
+--modules=file:///calico/modules.json \
```

Comment:
The modules.json file is not placed during the installation of Net modules, correct? Shouldn't there be a step that describes how to download this file?